From ae80bf833e48a6202f44a935a68083ae52ad3824
From: Ashok Rao 
Date: Sat, 11 Sep 2010 22:14:48 +1000
Subject: [PATCH] chat: Allow TIMEOUT value to come from environment variable

This allows the TIMEOUT value to be obtained from the environment
using the $ENVVAR syntax.

Signed-off-by: Paul Mackerras <paulus@samba.org>
---
 chat/chat.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/chat/chat.c b/chat/chat.c
index 0b88c27..710dba9 100644
--- a/chat/chat.c
+++ b/chat/chat.c
@@ -1167,6 +1167,7 @@ register char *s;
 
     if (timeout_next) {
 	timeout_next = 0;
+	s = clean(s, 0);
 	timeout = atoi(s);
 	
 	if (timeout <= 0)
-- 
1.7.1

